### ISSUE TITLE
[CI/CD] Improve docker image

### ISSUE THREAD

**Issue by MarinHadzhiev** [SAP/xsk#1239]
At the moment the size of the produced docker image is around 1.6 GB, we need to evaluate different options to improve here. Possible points for improvements are:

1. switch to Distroless base image
2. remove what we do not need from NEO SDK

---
### Target

1. Change base to be alpine
2. Change JVM to be [sapmachine 17](https://github.com/SAP/SapMachine)

**Comment by alexkuklin** [comment]
Current size breakdown is:

- image tomcat is built on: 124MB 
- tomcat base image:  327MB (+303MB)
- jdk: +321MB
- compiled WAR: 277MB

going distroless we can get something about 100MB at most and get management issues for updates and vulnerabilities. I highly doubt it is worth effort.

**Comment by alexkuklin** [comment]
Following up discussion on Thursday.

First, currently we are using 2y old tomcat8 image which is even not checked for log4j vulnerability, as per dockerhub info.

I built it over up-to-date tomcat9 image, particularly, tomcat:9.0.58-jre11-openjdk-slim-bullseye, 2 days old, without additional jdk.

With that image we have the following breakdown:
- 80.4MB base system
- +168MB tomcat system
- +250MB xsk WAR

Standalone xsk instance successfully started and I logged in without any problems with that setup. No idea though how to test it in further for now.
